cudamapper/src/overlapper_triggered.hpp

```diff
@@ -33,12 +33,10 @@ class OverlapperTriggered : public Overlapper
     /// Uses a dynamic programming approach where an overlap is "triggered" when a run of
     /// Anchors (e.g 3) with a score above a threshold is encountered and untriggerred
     /// when a single anchor with a threshold below the value is encountered.
-    /// \param overlaps Output vector into which generated overlaps will be placed
-    /// \param anchors vector of anchors sorted by query_read_id -> target_read_id -> query_position_in_read -> target_position_in_read (meaning sorted by query_read_id, then within a group of anchors with the same value of query_read_id sorted by target_read_id and so on)
-    /// \param index_query
-    /// \param index_target
+    /// \param fused_overlaps Output vector into which generated overlaps will be placed
```

Comment:
Maybe specify that names will not be updated and that `Overlapper::update_read_names()` has to be called? Not sure if this should be specified in `param` or general functions description

Reply:
added some description to both function description and in `param`